mkt/developers/views.py

```diff
@@ -373,12 +374,32 @@ def content_ratings_edit(request, addon_id, addon):
     }
     request.session.modified = True
 
-    return render(request, 'developers/apps/ratings/ratings_edit.html',
-                  {'addon': addon,
-                   'app_name': get_iarc_app_title(addon),
-                   'form': form,
-                   'company': addon.latest_version.developer_name,
-                   'now': datetime.now().strftime('%Y-%m-%d %H:%M:%S')})
+    ctx = {
+        'addon': addon,
+        'app_name': get_iarc_app_title(addon),
+        'form': form,
+        'company': addon.latest_version.developer_name,
+        'now': datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    }
+
+    if waffle.switch_is_active('iarc-upgrade'):
+        iarc_request_id = IARCRequestID.objects.filter(app_id=addon)
+        if iarc_request_id.exists():
```

Comment:
ah right i forgot since i needed to create a uuid in case it doesn't exist, but get_or_create takes create_args/extra_args right?